web/src/components/avatar/styles/avatar.ts

```diff
@@ -0,0 +1,66 @@
+import { JssStyle } from 'jss'
+import { createUseStyles } from 'react-jss'
+
+import tokens from '../../../tokens'
+import { parseToken } from '../../../utils'
+
+export const onScaleIndicationSize = (avatarSize: string): number =>
+  -parseToken(avatarSize) / 10 + 2
+
+export const useAvatarStyle = createUseStyles(
+  (theme: typeof tokens): Record<string, JssStyle> => ({
+    wrapper: {
+      display: 'flex',
+      position: 'relative',
+      padding: 0,
+      border: 0,
+      backgroundColor: 'transparent',
+      borderRadius: '50%',
+      pointerEvents: ({ isUnclickable }) => (isUnclickable ? 'none' : 'auto'),
+      width: ({ size }) =>
+        size === 'small' ? theme.sizeAvatarSmall : theme.sizeAvatarMedium,
+      height: ({ size }) =>
+        size === 'small' ? theme.sizeAvatarSmall : theme.sizeAvatarMedium,
+      cursor: ({ isDisabled }) => (isDisabled ? 'auto' : 'pointer'),
+      transition: 'all 0.25s ease-in-out',
+      '&:hover': {
+        transform: ({ isInactive, isDisabled }) =>
+          isInactive || isDisabled ? 'none' : 'scale(1.1)',
+        '& > div:nth-child(2)': {
+          transform: ({ isInactive, isDisabled }) =>
+            isInactive || isDisabled ? 'none' : 'scale(0.91)',
```

Comment:
0.91?

Reply:
Yeah, as `scale: 1.1` scales children too, and by design, we need to keep status/indicator of the same size for hovered/normal state. That's why I applied 0.91 here, to keep indicator of the same size on hover effect

Reply:
We should move the transform scale to the child (circle) then we don't need to scale back the other item!